src/components/form/FormFields.jsx

```diff
@@ -25,6 +33,27 @@ const FieldError = memo(({ error, text }) => {
  */
 export const FormFields = ({ value, field, error, register, control }) => {
   const name = field.name
+  const { watch } = useFormContext()
+  const [address, userInfo] = useUserStore(
+    (st) => [st.address, st.userInfo],
+    shallow
+  )
+  const [showVisualizer, setShowVisualizer] = useState(false)
+  const [audioBlob, setAudioBlob] = useState(null)
+  const visualizerRef = useRef(null)
+  const AUDIO_MIME_TYPES = [
```

Comment:
maybe it's better to put this with the rest of the types on https://github.com/teia-community/teia-ui/blob/main/src/constants.ts ?